app/views/stats/index.html.slim

```diff
@@ -46,6 +46,11 @@
           h4.card-title.mb-3 RDV par type (#{@stats.rdvs_for_default_range.count})
           = column_chart add_query_string_params_to_url(rdvs_stats_path, by_location_type: true, departement: @departement), stacked: true
 
+      .card.mb-5
+        .card-body
+          h4.card-title.mb-3 RDV par statut (#{@stats.rdvs_for_default_range.count})
```

Comment:
This count is incorrect as additional filters are applied.

Reply:
👍 good catch, I'll remove it!

Reply:
You can remove it as the appropriate one would not be very useful as data is normally on a per week basis.

Reply:
:dash: